package.json

```diff
@@ -80,17 +80,17 @@
     "zustand": "^3.7.2"
   },
   "devDependencies": {
-    "@commitlint/cli": "^17.6.7",
-    "@commitlint/config-conventional": "^17.6.7",
+    "@commitlint/cli": "^17.7.1",
+    "@commitlint/config-conventional": "^17.7.0",
     "@jest/types": "^29.6.1",
-    "@playwright/test": "^1.36.1",
-    "@swc/core": "^1.3.68",
-    "@swc/jest": "^0.2.27",
+    "@playwright/test": "^1.37.1",
+    "@swc/core": "1.3.68",
```

Comment:
You shouldn't need to remove the ^ but instead revert the upgrade of those packages in the lock file.

Reply:
if any other pr adds a new dependency and does yarn install which overrites the yarn.lock. if the caret symbol is in the dependency it will try to find the latest version compatible with the current version. 

```✳️ @swc/core (1.3.68 → 1.3.75, patch)``` you can see they defined a patch which indicate backward compatible. but its still breaking things

if we keep caret. probably it will break things again! or i am missing anything?